src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+// var plotAttrs = require('../../plots/attributes');
+// var domainAttrs = require('../../plots/domain').attributes;
```

Comment:
:zap: those commented out lines.

Reply:
Done in e6d8525ab07c56ca10bdbac6762b06d1d57bf830